src/ConnectedMode/Suppressions/IssuesFilter.cs

```diff
@@ -28,6 +28,10 @@ namespace SonarLint.VisualStudio.ConnectedMode.Suppressions
 {
     public interface IIssuesFilter
     {
+        /// <summary>
+        /// Returns the list of supplied objects that match the filter
```

Comment:
Miscellaneous: I added a comment because I got confused about what the interface was returning